contracts/eos/BancorConverter/BancorConverter.hpp

```diff
@@ -90,11 +91,6 @@ CONTRACT BancorConverter : public eosio::contract { /*! \endcond */
                  * @brief account name of contract for voting and staking
                  */
                 name staking;
-
-                /*! \cond DOCS_EXCLUDE */
-                uint64_t primary_key() const { return "settings"_n.value; }
```

Comment:
no need to define the key here?

Reply:
No, singleton tables don't need a primary key, it's essentially 1 row, so it always assumes to fetch the first row as it's primary key.

Reply:
oh gotcha, cool